### DISCUSSION TITLE
Resource requirements for Honeypot?

### DISCUSSION THREAD

**Discussion by RagnarSTS** [Security-Onion-Solutions/securityonion#7506] [General] [answered]
Looking forward to the honeypot node integration.  I have a question on the installation sequence and hardware requirements.
Currently can run a centos machine with canary and have it fire off an email or other alert whenever it gets poked at and I can do it with something like 6G of drive space and 2G of memory. 
I've even run smaller honeypots that function as more of a tripwire with only 300M of memory. 
SO generally requires much larger hardware to be allocated.  Is there a way we can shim it down to something more reasonable?  Would be neat if I could get out of band communications on one interface while I have a small honeynet of 10 fake machines without using a ton of resources.

**Comment by dougburks** [comment, marked as answer]
Resource requirements are listed here:
https://docs.securityonion.net/en/2.3/architecture.html#intrusion-detection-honeypot-idh-node

You may be able to get by with less but those specs should work well.

**Reply by Sazerak** [reply]
I know this is an old post, but I had the same question and noticed that the answer is not actually in the Architecture section, but rather the Hardware Requirements section:
https://docs.securityonion.net/en/2.3/hardware.html#intrusion-detection-honeypot-idh-node
